kubernetes/samples/charts/util/setupLoadBalancer.sh

```diff
@@ -74,18 +82,22 @@ function createTraefik() {
   createNameSpace $TSPACE
   echo "Creating Traefik operator on namespace ${TSPACE}" 
 
-  if [ "$(helm search stable/traefik | grep traefik |  wc -l)" = 0 ]; then
-    echo "Add K8SGoogle chart repository"
-    helm repo add stable https://kubernetes-charts.storage.googleapis.com
+  if [ "$(helm search repo traefik/traefik | grep traefik |  wc -l)" = 0 ]; then
+    # https://containous.github.io/traefik-helm-chart/
+    # https://docs.traefik.io/getting-started/install-traefik/
+    echo "Add containous.github.io chart repository"
+    helm repo add traefik https://containous.github.io/traefik-helm-chart
     helm repo update
   else
-    echo "K8SGoogle chart repository is already added."
+    echo "Containous chart repository is already added."
```

Comment:
People will wonder what contianous is... it would be better to say "Traefik chart repository...."

Reply:
Earlier description says K8sGoogle so I followed the same pattern.  I moved it back to Traefik